requirements/requirements.txt

```diff
@@ -1,13 +1,13 @@
 # Framework must-haves
 numpy>=1.16.4
-torch>=0.4.1
+torch>=1.2.0
 torchvision>=0.2.1
 imageio
 opencv-python
 scikit-image>=0.14.2
 tqdm>=4.29.1
 PyYAML
-tensorboardX>=1.8
+tensorboard
```

Comment:
Does this package even exist? And is it necessary?

Reply:
Tensorboard package is needed for PyTorch native one
https://pytorch.org/docs/stable/tensorboard.html
If you're speaking of tensorboardX, i guess it isn't necessary to keep it.